zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
+| getpeerinfo                 | YES, partially.    | The explorer takes all the fields and add them to a `nodes`  array. The following extra fields are needed in Zebra:<br>`subver`<br>`synced_blocks`                                       | [Zcash](https://zcash.github.io/rpc/getpeerinfo.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.GetBlockTemplateRpc.html#tymethod.get_peer_info)          | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/nodes/nodes_warmer.ex#L14) [called by](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/live/nodes_live.ex#L17)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| getrawtransaction           | YES, partially.    | The `blockhash` argument is not used in the explorer which is good but at least the following extra fields are needed in the response:<br>`txid`<br>`time` (from block header)<br>`size`<br>`vout`<br>tx type: many transparent and shielded fields<br>We'll need to add `tx.size` to the existing database and non-finalized state indexes.               | [Zcash](https://zcash.github.io/rpc/getrawtransaction.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.Rpc.html#tymethod.get_raw_transaction)              | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/transactions/transaction_cache.ex#L30) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/search_controller.ex#L15) - [3](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/transaction_controller.ex#L5) [called by](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/live/recent_transactions_live.ex#L19) - [4](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/transaction_controller.ex#L12)                                                                                                     |
```

Comment:
Changed to the RPC fields used by the block explorer, removed the ones it calculates itself. Unfortunately we need a lot of fields for the explorer to do the transaction type.